src/Geocoder/Geocoder.php

```diff
@@ -47,6 +47,8 @@ class Geocoder {
 		'region' => null, // For GoogleMaps provider
 		'ssl' => true, // For GoogleMaps provider
 		'apiKey' => '', // For GoogleMaps provider,
+		'addresssFormat' => '%S %n, %z %L',
```

Comment:
addresssFormat? oops

Reply:
hahah. good catch. do you wanna revert the merge so I can get that.

Reply:
tbh, i think I added that before I figured out the whole `if(!file_exists(...))` that's a really neat way to cut API calls out of the test suite. I am going to start using that!!

Reply:
I fixed in master and also streamlined config in https://github.com/dereuromark/cakephp-geo/commit/fbadfe071cc29a0f84d481069f19b070897fd74c